std_msgs/Header header
vmr_ros_pkg/Exception[] exceptions